pkg/framework/plugins/defaultevictor/defaultevictor.go

```diff
@@ -143,6 +145,38 @@ func New(args runtime.Object, handle frameworktypes.Handle) (frameworktypes.Plug
 		})
 	}
 
+	if defaultEvictorArgs.MinReplicas > 1 {
+		indexName := "metadata.ownerReferences"
+		indexer, err := getPodIndexerByOwnerRefs(indexName, handle)
+		if err != nil {
+			return nil, err
+		}
+		ev.constraints = append(ev.constraints, func(pod *v1.Pod) error {
+			if len(pod.OwnerReferences) == 0 {
+				return nil
+			}
+
+			if len(pod.OwnerReferences) > 1 {
+				klog.V(5).InfoS("pod has multiple owner references which is not supported for minReplicas check", "size", len(pod.OwnerReferences), "pod", klog.KObj(pod))
+				return nil
+			}
+
+			ownerRef := pod.OwnerReferences[0]
+			objs, err := indexer.ByIndex(indexName, string(ownerRef.UID))
+			if err != nil {
+				return fmt.Errorf("unable to list pods for minReplicas filter in the policy parameter")
+			}
+
+			pods := podutil.ConvertToPods(objs, nil)
+			ownerReplicas := podutil.GetOwnerReplicasCount(pods, pod)
```

Comment:
`len(objs)` should be already sufficient for the check (assuming the informer already knows about `pod`).